.projenrc.js

```diff
@@ -1,7 +1,7 @@
 const { awscdk, javascript, DependencyType } = require("projen");
 const { workflows } = require("projen/lib/github");
 
-const CDK_VERSION = "2.60.0";
+const CDK_VERSION = "2.65.0";
```

Comment:
Changing the peer dependencies is a breaking change.

You can refer to https://github.com/cdklabs/cdk-monitoring-constructs/pull/303 for an example of how to handle this. Note:

- The commit message format
- The bumping of `majorVersion` in this file
- Updating the README to reflect the updated peer deps

Reply:
Thanks. I was indeed missing those steps. I've fixed it in the newest commit.

- [ ] The commit message format
- [ ] The bumping of majorVersion in this file
- [ ] Updating the README to reflect the updated peer deps